plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -341,7 +348,28 @@ public Instance startService(String namespace,
         }
     }
 
-    @Override
+    // sets the home directory for a user defined with CHE_USER
+    private void changeWorkingDirOwner(String containerId, LineConsumer machineLogger) throws MachineException {
+        String prepareCmd = "CHE_USER=`id -un`;"
+                + "sudo chown -R ${CHE_USER}:docker /home/user;"
```

Comment:
@benoitf do you agree that $HOME does not correspond to /home/user here right? By the way `/home/user/`